Makefile

```diff
@@ -222,3 +235,26 @@ swagger-docs: ## preview the API documentation
 		-e 'REDOC_OPTIONS=hide-hostname="true" lazy-rendering' \
 		-p $(SWAGGER_DOCS_PORT):80 \
 		bfirsh/redoc:1.6.2
+
+.PHONY: buildx
+ifeq ($(BUILDX), bundles/buildx)
+buildx: bundles/buildx # build buildx cli tool
+else
+buildx:
+endif
+
+bundles/buildx: GOOS ?= $(shell go env GOHOSTOS || uname | awk '{print tolower($$0)}' || echo windows)
+bundles/buildx: GOARCH ?= $(shell go env GOHOSTARCH || echo amd64)
+bundles/buildx: DOCKER_BUILDX_VERSION ?= v0.3.0
```

Comment:
Looks like we now define defaults in two locations; a default inside the Dockerfile (`ARG BUILDX_COMMIT=v0.2.2`), and one here, in the Makefile (also see my comment below)

Reply:
Problem is I'm using this to tag the image.

Reply:
I've updated the Dockerfile to just default to master.